Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -901,9 +901,12 @@ - (void)validateActiveTargets {
     XCTAssertEqual(actual.snapshot_version(), targetData.snapshot_version());
     XCTAssertEqual(actual.resume_token(), targetData.resume_token());
     if (targetData.expected_count().has_value()) {
-      HARD_ASSERT(actual.expected_count().has_value(),
-                  "Actual Target data doesn't have expected count.");
-      XCTAssertEqual(actual.expected_count().value(), targetData.expected_count().value());
+      if (!actual.expected_count().has_value()) {
+        XCTFail("");
+      } else {
+        XCTAssertTrue(actual.expected_count().has_value());
```

Comment:
Can you remove this line:

```
XCTAssertTrue(actual.expected_count().has_value());
```

Isn't this verified in the "if" condition on line 904?